### PR TITLE
[contribute] New contributor landing page!

### PR DESCRIPTION
**Pull request by sarah11918**
Adds a new landing page in docs for our various contributing guides for better discoverability. 

Notable:
- links to existing ALLCAPSGUIDES.md files
- Re-uses FacePile in prep for our full list of GitHub contributors being eventually removed from the landing page with the new Docs theme to come.

Includes:
- Right sidebar menu item for "Contributor Guides"
- new translation key in `ui.ts`

TBD/out of scope:
- do the individual GUIDES.md files eventually live as docs pages?

contribute

### CODE REVIEW

File: src/content/docs/en/contribute.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks! 
+
+Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
+
+## Contributor manuals
```
Comment (ElianCodes) [inline src/content/docs/en/contribute.mdx:23]: Maybe this is just me, but I think it's strange to address "Contributor manuals" & "Community manuals" and then only make a list of "guides".

It's just wording, and you're the OG of Astro docs, so feel free to ignore 😅

Comment (Jutanium) [reply src/content/docs/en/contribute.mdx:23]: I agree with Elian here - I prefer "Contributor Guides"

Comment (sarah11918) [reply src/content/docs/en/contribute.mdx:23]: Changing it!

File: src/content/docs/en/contribute.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+We have several guides with information on how to get started!
+
+- [Main Contributing Guide](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - Please consult this first!
+- [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) - Tips for writing in Astro Docs style.
+- [Translating Guide](https://github.com/withastro/docs/blob/main/TRANSLATING.md) - Information on how to help translate Astro Docs into your language.
+- [Recipe Guide](https://github.com/withastro/docs/blob/main/RECIPES.md) - Learn about writing and contributing a recipe.
+
+## Community manuals
```
Comment (sarah11918) [inline src/content/docs/en/contribute.mdx:32]:
```suggestion
## Contributor guides

We have several guides with information on how to get started!

- [Main Contributing Guide](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) - Please consult this first!
- [Writing Guide](https://github.com/withastro/docs/blob/main/WRITING.md) - Tips for writing in Astro Docs style.
- [Translating Guide](https://github.com/withastro/docs/blob/main/TRANSLATING.md) - Information on how to help translate Astro Docs into your language.
- [Recipe Guide](https://github.com/withastro/docs/blob/main/RECIPES.md) - Learn about writing and contributing a recipe.

## Community guides
```

**Comment by sarah11918** [comment]
CI tests won't run, so I'm letting this one sit a while.

Will also need @Yan-Thomas to check my work, and I actually reused the book SVG in the new menu item. If someone wants to add a different one (either to blog or contributor guides) so that we don't have a dupe, please feel free!

File: src/content/docs/en/contribute.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+title: Contribute to the Docs
+description: A guide to contributing to Astro Docs.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+We welcome contributions of any size and contributors of any skill level. As an open source project, we believe in giving back to our contributors. We are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
```
Comment (Yan-Thomas) [inline src/content/docs/en/contribute.mdx:11]: For consistency's sake, I'd use "open-source" here! 
```suggestion
We welcome contributions of any size and contributors of any skill level. As an open-source project, we believe in giving back to our contributors. We are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
```

File: src/content/docs/en/contribute.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Ways to Contribute
+
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks!
```
Comment (Yan-Thomas) [inline src/content/docs/en/contribute.mdx:19]: Serial comma
```suggestion
You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions, and participating in our "Pinned" issue maintenance tasks! 
```

File: src/content/docs/en/contribute.mdx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+## Ways to Contribute
+
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks! 
+
+Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
```
Comment (Yan-Thomas) [inline src/content/docs/en/contribute.mdx:21]: deja vú comma
```suggestion
Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions, or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
```

**Review by Yan-Thomas — COMMENTED** [review]
I think we chatted about adding info on who you can reach out to if you want to contribute and need help from Astro Docs Maintainers, maybe here's a good place to add this? Non-blocking, could very well be a followup PR.

Added a few suggestions, after these LGTM!

Edit: I'm not sure about the icon, maybe the PR one works https://primer.style/design/foundations/icons/git-pull-request-24?

**Comment by sarah11918** [comment]
Waiting for Yan to fix my.... RightSidebar!  :sweat_smile:

**Review by Jutanium — APPROVED** [review]
Now that it's "Guide"s, LGTM!

**Pull request merged**